### DISCUSSION TITLE
Try image predition on colab

### DISCUSSION THREAD

**Discussion by longtimenoseeyou** [autogluon/autogluon#2248] [Q&A]
![1666613653560](https://user-images.githubusercontent.com/50489319/197524493-13e02eff-cc8d-4693-9d8c-c592f93d1cc9.png)
how do I tackle it

**Comment by Innixma** [comment]
Please provide a reproducible example colab link, the screenshot provided is not sufficient to help.

**Reply by longtimenoseeyou** [reply]
https://colab.research.google.com/drive/1Wnk2dld2kOKJCrkXUe4gKK1879_UJnRR#scrollTo=lW8EnD-ZvIyX
